tools/svg2qvg/CMakeLists.txt

```diff
@@ -46,3 +46,28 @@ target_link_libraries(${target} PRIVATE Qt::Svg)
 set_target_properties(${target} PROPERTIES FOLDER tools)
 
 install(TARGETS ${target})
+
+# packaging
+set(PACKAGE_TARGET_FILENAME QSkinnyToolsTargets.cmake)
+
+set_target_properties(${target} PROPERTIES EXPORT_NAME Svg2Qvg)
+set_target_properties(${target} PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
@uwerat I needed this in order to get svg2qvg running in my integration test on Mac. Does this cause problems from a user's perspective?